docs/monitor_troubleshoot.rst

```diff
@@ -0,0 +1,209 @@
+Troubleshoot Monitoring
+========================
+
+
+This document describes steps that need to be done to troubleshoot monitoring problems when using `Grafana/Prometheus`_ monitoring tool.
+
+..  _`Grafana/Prometheus`: /monitoring_apis/
+
+Problem
+~~~~~~~
+
+.. contents::
+   :depth: 1
+   :local:
+
+A Container Fails To Start
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When running ``./start-all.sh`` a container can fail to start. For example you can see the following error message:
+
+.. code-block:: shell
+
+   Wait for Prometheus container to start........Error: Prometheus container failed to start
+
+
+Should this happen, check the Docker logs for more information.
+
+.. code-block:: shell
+
+   docker logs aprom
+
+Usually the reason for the failure is described in the logs.
+
+Files And Directory Permissions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+
+.. note::
+
+   Avoid running Docker containers as root.
+
+The preferred way of running the container is using a non root user.
+See the `monitoring`_ Docker post-installation section.
+
+.. _`monitoring`: /operating-scylla/monitoring/3.4/monitoring_stack/#docker-post-installation 
```

Comment:
```suggestion
.. _`monitoring`: /monitoring_stack/#docker-post-installation 
```

Need to check
All link should be update to this project and not include version